src/language/mixins.md

```diff
@@ -26,10 +26,17 @@ class Maestro extends Person [!with Musical, Aggressive, Demented!] {
 }
```

Comment:
I think we also have to update the lead-in up in lines 9-10 to something like "Mixins are a way of defining code that can be reused in multiple class hierarchies" (i.e. not talk about "a class's code")? cc @leafpetersen